src/modules/dpos/validator/components/delegateProfile/delegateProfile.css

```diff
@@ -45,6 +57,38 @@
     & button {
       grid-column: 1/5;
       margin-top: -15px;
+      border-top: none;
+    }
+  }
+
+  & .accountSummary {
+    display: flex;
+    flex-direction: column;
+    margin-left: 8px;
+
+    & p {
+      @mixin headingSmall;
+
+      --content-font: 'gilroy';
```

Comment:
@eniolam1000752 Using the variable `--heading-font` might be a better way to do this rather than declaring a new variable